CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ A breaking change will get clearly marked in this log.
 
 ## Unreleased
 
+## [v10.4.1](https://github.com/stellar/js-stellar-sdk/compare/v10.4.0...v10.4.1)
+
+### Add
+
+- Enables signing in service workers using FastSigning
+
+### Update
+
+- Bumps `stellar-base` version to
+  [v8.2.2](https://github.com/stellar/js-stellar-base/releases/tag/v8.2.2) to
+  include latest fixes.
```

Comment:
I hate to nitpick here, sorry, but let's stick with the existing format and keep the text unwrapped, and also keep `stellar-base` changes isolated to that changelog entry!

```suggestion

## [v10.4.1](https://github.com/stellar/js-stellar-sdk/compare/v10.4.0...v10.4.1)

### Update

- Bumps `stellar-base` version to [v8.2.2](https://github.com/stellar/js-stellar-base/releases/tag/v8.2.2) to include latest fix: enabling fast signing in service workers ([#806](https://github.com/stellar/js-stellar-sdk/pull/806)).
```

(sorry I'm really nitty about my precious changelogs :joy:)